compiler/src/model/utils.ts

```diff
@@ -648,6 +635,34 @@ export function hoistRequestAnnotations (
       const docUrl = docIds.find(entry => entry[0] === value.trim())
       assert(jsDocs, docUrl != null, `The @doc_id '${value.trim()}' is not present in _doc_ids/table.csv`)
       endpoint.docUrl = docUrl[1]
+    } else if (tag === 'availability') {
+      // The @availability jsTag is different than most because it allows
+      // multiple values within the same docstring, hence needing to parse
+      // the values again in order to preserve multiple values.
+      const jsDocsMulti = parseJsDocTagsAllowDuplicates(jsDocs)
+      const availabilities = parseAvailabilityTags(jsDocs, jsDocsMulti.availability)
+
+      // Apply the availabilities to the Endpoint.
+      for (const availabilityName in availabilities) {
+        const availabilityValue = availabilities[availabilityName]
```

Comment:
```suggestion
      for (const [availabilityName, availabilityValue] of Object.entries(availabilities)) {
```
Nitpick, just drops the object iteration logic down to one line.